merfi/iso.py

```diff
@@ -36,7 +39,14 @@ def parse_args(self):
         self.source = util.infer_path(parser.unknown_commands)
         self.check_dependency()
         self.make_iso()
+        self.make_sha256sum()
 
     def make_iso(self):
         cmd = ['genisoimage', '-r', '-o', self.output, self.source]
         util.run(cmd)
+
+    def make_sha256sum(self):
+        with open(self.output, 'rb') as isof:
```

Comment:
this approach is OK as long as the file to read is not big otherwise it is read fully into memory. An alternative approach would be reading smaller chunks while having the same result in the end:

    def make_sha256sum(self):
        chsum = hashlib.sha256()
        with open(self.output, "rb") as f:
            for chunk in iter(lambda: f.read(4096), b""):
                chsum.update(chunk)
        digest = chsum.hexdigest()
        with open(self.output + '.SHA256SUM', 'w') as shaf:
            shaf.write("%s  %s\n" % (sha256sum, os.path.basename(self.output)))